ajaxtest/ajax_info.txt

```diff
@@ -0,0 +1,4 @@
+<h1>AJAX</h1>
```

Comment:
I don't think you're using this file anymore. You should get rid of it to avoid confusing anyone who will use your example as a reference in the future.